src/components/ManageListForms/EditItemForm.jsx

```diff
@@ -0,0 +1,179 @@
+		setFormData((prevFormData) => ({
+			...prevFormData,
+			daysUntilNextPurchase: value,
+		}));
+	};
+
+	return (
+		<form
+			className="flex flex-col items-center gap-8 max-w-[400px] text-black"
+			onSubmit={(event) => handleSubmit(event)}
+		>
+			<div className="flex flex-col gap-2 w-full">
+				<label htmlFor="itemName" className="text-md font-medium">
+					Item name{' '}
+				</label>
+				<Input
+					type="text"
+					name="itemName"
+					id="itemName"
+					placeholder="Enter item name"
+					required
+					value={formData.itemName}
+					onChange={handleChange}
+				/>
+			</div>
+			<div className="flex flex-col gap-2 w-full">
+				<label htmlFor="daysUntilNextPurchase" className="text-md font-medium">
+					How soon would you like to buy this again?
+				</label>
+				<RadioGroup
+					onValueChange={handleRadioChange}
+					className="flex my-2 items-center justify-center gap-4"
+					id="daysUntilNextPurchase"
+				>
+					<div className="flex flex-col items-center justify-center rounded-xl border border-light-pink gap-4 w-24 h-24 sm:w-28 sm:h-28 shadow-bottom-right transition-transform duration-200 ease-in-out transform active:scale-95">
+						<RadioGroupItem
+							value="7"
+							id="soon"
+							name="timeFrame"
+							className="border border-soon text-soon"
+						/>
+						<label htmlFor="soon" className="font-semibold text-sm">
+							Soon
+						</label>
+					</div>
+					<div className="flex flex-col items-center justify-center rounded-xl border border-light-pink gap-4 w-24 h-24 sm:w-28 sm:h-28 shadow-bottom-right transition-transform duration-200 ease-in-out transform active:scale-95">
+						<RadioGroupItem
+							value="14"
+							id="kind-of-soon"
+							name="timeFrame"
+							className="border border-kind-of-soon text-kind-of-soon"
+						/>
+						<label htmlFor="kind-of-soon" className="font-semibold text-sm">
+							Kind of soon
+						</label>
+					</div>
+					<div className="flex flex-col items-center justify-center rounded-xl border border-light-pink gap-4 w-24 h-24 sm:w-28 sm:h-28 shadow-bottom-right transition-transform duration-200 ease-in-out transform active:scale-95">
+						<RadioGroupItem value="30" id="not-of-soon" name="timeFrame" />
+						<label htmlFor="not of soon" className="font-semibold text-sm">
+							Not soon
+						</label>
+					</div>
+				</RadioGroup>
+			</div>
+			<div className="flex flex-col items-start gap-2 w-full">
+				<label htmlFor="quantity" className="text-md font-medium">
+					Quantity
+				</label>
+				<div className="flex items-start">
+					<Input
+						type="number"
+						name="itemQuantity"
+						id="quantity"
+						required
+						value={formData.itemQuantity}
+						onChange={handleChange}
+						min={1}
+						max={100}
+					/>
+				</div>
+			</div>
+			<div className="flex w-full">
+				<Button
+					type="submit"
+					className="bg-primary-pink text-white rounded-xl w-full hover:bg-primary-pink hover:bg-opacity-75 text-sm"
```

Comment:
The text should be black, just because all other pink buttons have black text

Reply:
I think leaving it white is a better contrast even for the delete dialog what do you think? i lowered the opacity on hover to work with white see the new changes. Also i think leaving pencil and trash can green in dark mode looks good.

Reply:
It has to to with accessibility the the white on pink is harder to read. I saw you changed it on some of the other buttons, if you prefer to stick to white and pink, do white and pink but it will need be applied across all the pink buttons for consistency.
And the hover back on the icons like in the single page `transform hover:scale-110 transition-transform duration-150 sm:hover:scale-125`  to have a more visual feedback to the user.